src/routes/v3/users.rs

```diff
@@ -65,23 +65,12 @@ pub async fn projects_list(
     let id_option = User::get(&info.into_inner().0, &**pool, &redis).await?;
 
     if let Some(id) = id_option.map(|x| x.id) {
-        let user_id: UserId = id.into();
-
-        let can_view_private = user
-            .map(|y| y.role.is_mod() || y.id == user_id)
-            .unwrap_or(false);
-
         let project_data = User::get_projects(id, &**pool, &redis).await?;
 
-        let response: Vec<_> =
-            crate::database::Project::get_many_ids(&project_data, &**pool, &redis)
-                .await?
-                .into_iter()
-                .filter(|x| can_view_private || x.inner.status.is_searchable())
-                .map(Project::from)
-                .collect();
-
-        Ok(HttpResponse::Ok().json(response))
+        let projects: Vec<_> =
+            crate::database::Project::get_many_ids(&project_data, &**pool, &redis).await?;
```

Comment:
this change ported over from extracted_versions, makes the user route consistent with the other routes that only let you see some projects, and allosw the filter visible projeccts test down below to pass